rules/link-users-by-email-with-metadata.js

```diff
@@ -32,6 +32,11 @@ function (user, context, callback) {
   const userApiUrl = auth0.baseUrl + '/users';
   const userSearchApiUrl = auth0.baseUrl + '/users-by-email';
 
+  // currentUser : The user that just logged in, same as "user"
```

Comment:
hmm this code does not use `currentUser` right? should we drop this?

Reply:
Good catch, ya. I've merged your suggestion and removed it.